### PR TITLE
Add Icons and Fix Leaked Icons

### PR DESCRIPTION
**Pull request by lolfuljames**
- Hide icons in infowindow from showing before infowindow is generated
- Remove iconify class from css and use data-width and data-height instead to control size.
- Change collapse/expand/submit to use icons from iconify
- Change directions from png to SVG from iconify

Demo: https://jtan-sps-summer20.appspot.com/

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -30,26 +30,31 @@
...
-          <span class="iconify" data-icon="logos:telegram" data-inline="false"></span>
+          <span class="iconify" data-icon="logos:telegram" data-inline="false" data-width="35px" data-height="35px"></span>
         </a>
       </div>
     </div>
     <div id="right-tab">
       <div id="pac-card">
         <h1>EatGoWhere</h1>
-        <a id="resize-nav-button">Collapse</a>
+        <a id="resize-nav-button">
```
Comment (jieliangang) [inline src/main/webapp/index.html]: Can we change this to a button instead of a link? makes more sense

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -114,7 +114,11 @@ let selectedIndex;
 
 function updateRestaurantList(restaurants) {
   let resContainerElement = document.getElementById("result-container");
-  resContainerElement.style.display = "block";
+
+  resContainerElement.classList.remove("hide");
```
Comment (jieliangang) [inline src/main/webapp/script.js:119]: nice work fixing this. however note that im going to remove this after shifting the icons to the restaurant section.

**Review by jieliangang — APPROVED** [review]
Great work. UI looks cleaner now after replacing the words with icon.

**Review by jieliangang — CHANGES_REQUESTED** [review]
Please fix merge conflict.

File: src/main/webapp/index.html (hunk 1 above)
Comment (lolfuljames) [reply src/main/webapp/index.html]: Thanks for pointing it out, no idea why I started with that

**Review by jieliangang — APPROVED** [review]
lgtm

**Pull request merged**